src/systems/particle_emitter/ParticleEmitter.hh

```diff
@@ -76,25 +76,15 @@ namespace systems
   /// `<color_start>`: Sets the starting color for all particle emitted.
   ///                  The actual color will be interpolated between this color
   ///                  and the one set under <color_end>.
-  ///                  Input is specified as a string (should be lower-case),
-  ///                  which could be one of the following:
-  ///                   * black
-  ///                   * red
-  ///                   * green
-  ///                   * blue
-  ///                   * yellow
-  ///                   * magenta
-  ///                   * cyan
-  ///                   * white
-  ///                  Default color is white.
+  ///                  Color::White is the default color for the particles
+  ///                  unless a specific function is used.
```

Comment:
Can we give example input so that users know what `specific function` means? For example, I am assuming that input for red would be `<color_start>1 0 0</color_start>`.

We should also add this information in to the `color_end` documentation below.

Reply:
https://github.com/ignitionrobotics/ign-gazebo/pull/615/commits/4f64296d87f3f3b37bf22b763a0d61dde0b78355